res/ai/DefaultAIPlayer/DefaultAIPlayer.lua

```diff
@@ -897,7 +897,7 @@ function OnMinute(number)
 	-- on xx:06 check if there is an unsatisfiable ad planned for this
 	-- hour
 	if minute == 6 then
-		local task = getAIPlayer().TaskList[_G["TASK_SCHEDULE"]]
+		local task = player.TaskList[_G["TASK_SCHEDULE"]]
```

Comment:
```
-- figure is now in this room
function OnEnterRoom(roomId)
	roomId = tonumber(roomId) --incoming roomId is "string"

	if (aiIsActive) then
		getAIPlayer():OnEnterRoom(roomId)

		-- when visiting the boss or betty, update sammy information
		if roomId == TVT.ROOM_BOSS_PLAYER_ME then
			--debugMsg("Visiting my boss", true)

			getAIPlayer().currentAwardType = TVT.bo_GetCurrentAwardType()
			getAIPlayer().currentAwardStartTime = tonumber(TVT.bo_GetCurrentAwardStartTimeString())
			getAIPlayer().currentAwardEndTime = tonumber(TVT.bo_GetCurrentAwardEndTimeString())

			getAIPlayer().nextAwardType = TVT.bo_GetNextAwardType()
			getAIPlayer().nextAwardStartTime = tonumber(TVT.bo_GetNextAwardStartTimeString())
			getAIPlayer().nextAwardEndTime = tonumber(TVT.bo_GetNextAwardEndTimeString())
		end
	end
```
Koennte man in dem Schritt wohl auch gleich auf `player` umstellen (Funktionsaufrufe einsparen)